homeassistant/components/fritz/common.py

```diff
@@ -386,64 +379,20 @@ async def async_trigger_firmware_update(self) -> bool:
 
     async def async_trigger_reboot(self) -> None:
         """Trigger device reboot."""
-        await self.hass.async_add_executor_job(
-            self.connection.call_action, "DeviceConfig1", "Reboot"
-        )
+        await self.connection.reboot()
```

Comment:
this needs to be wrapped into `hass.async_add_executor_job()`

Reply:
Fixed